debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/PostgresReplicationConnection.java

```diff
@@ -119,29 +119,30 @@ protected void initPublication() {
         if (PostgresConnectorConfig.LogicalDecoder.PGOUTPUT.equals(plugin)) {
             LOGGER.info("Initializing PgOutput logical decoder publication");
             try {
-                try (Statement stmt = pgConnection().createStatement()) {
```

Comment:
I'm a bit curious: wouldn't it have sufficed to simple add `rs` to the `try` declaration next to `stmt`?

Reply:
In hindsight it would, the try-with-resources syntax with multiple resources isn't something I have used quite often and so I often forget it exists, so yes I could have simplified it as such and I can certainly change that if you'd prefer it that way.

Reply:
Actually I think from a diff perspective doing it that way looks more clear; I'll push that shortly.